src/pybind/mgr/cephadm/module.py

```diff
@@ -1062,6 +1074,94 @@ def _show_exporter_config(self) -> Tuple[int, str, str]:
         cfg = self._get_exporter_config()
         return 0, json.dumps(cfg, indent=2), ""
 
+    @orchestrator._cli_read_command('cephadm config-check ls')
+    def _config_checks_list(self, format: Format = Format.plain) -> HandleCommandResult:
+        """List the available configuration checks and their current state"""
+
+        if format not in [Format.plain, Format.json, Format.json_pretty]:
+            return HandleCommandResult(
+                retval=1,
+                stderr="Requested format is not supported when listing configuration checks"
+            )
+
+        if format in [Format.json, Format.json_pretty]:
```

Comment:
```suggestion
        if format != Format.plain:
```

No need to artificially exclude yaml here.